docs/_templates/external-data-v1.rst

```diff
@@ -81,7 +82,7 @@
    .. admonition:: External data
       :class: note external-data
    
-      This data product is developed by an external party, and is not a DEA product.
+      This data product is developed by an external party ({{ Data.external_provider }}), and is not a DEA product. The data hasn't been altered except to make it compatible with DEA's systems.
```

Comment:
Slight wording update to be less colloquial:

> The data **has not** been altered except to make it compatible with DEA's systems.

Reply:
Latest:

> This data product is produced by an external party ({{ Data.external_provider }}), and is not a DEA product. DEA provides this data without modifications except where needed to make it compatible with our systems.